Description: The examples in the cdk_display(3) man page are incorrect and
 don't work as expected (wrong colors, segfault, ...).
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=593283
Author: Loic Pefferkorn <loic-debianbts@loicp.eu>

Index: libcdk5-5.0.20060507/man/cdk_display.3
===================================================================
--- libcdk5-5.0.20060507.orig/man/cdk_display.3	2006-04-17 06:45:02.000000000 -0400
+++ libcdk5-5.0.20060507/man/cdk_display.3	2011-02-10 19:48:26.059539574 -0500
@@ -67,10 +67,13 @@
    screen = initscr();
    cdkscreen = initCDKScreen (screen);
 
+   /* Start CDK Colors */
+   initCDKColor();
+
    /* Set the labels up.		*/
-   mesg[0] = "</1>This line should have a yellow foreground and a blue background.<!1>";
-   mesg[1] = "</2>This line should have a white  foreground and a blue background.<!2>";
-   mesg[2] = "</3>This line should have a yellow foreground and a red  background.<!3>";
+   mesg[0] = "</31>This line should have a yellow foreground and a blue background.<!31>";
+   mesg[1] = "</05>This line should have a white  foreground and a blue background.<!05>";
+   mesg[2] = "</26>This line should have a yellow foreground and a red  background.<!26>";
    mesg[3] = "<C>This line should be set to whatever the screen default is.";
 
    /* Declare the labels.	*/
@@ -134,10 +137,13 @@
    screen = initscr();
    cdkscreen = initCDKScreen (screen);
 
+   /* Start CDK Colors */
+   initCDKColor();
+
    /* Set the labels up.  */
-   mesg[0] = "</B/1>Bold text            yellow foreground / blue background.<!1>";
-   mesg[1] = "</U/2>Underlined text      white  foreground / blue background.<!2>";
-   mesg[2] = "</K/3>Blinking text        yellow foreground / red  background.<!3>";
+   mesg[0] = "</B/31>Bold text            yellow foreground / blue background.<!31>";
+   mesg[1] = "</U/05>Underlined text      white  foreground / blue background.<!05>";
+   mesg[2] = "</K/26>Blinking text        yellow foreground / red  background.<!26>";
    mesg[3] = "<C>This line uses the screen default colors.";
 
    /* Declare the labels.  */
@@ -197,15 +203,18 @@
    CDKSCREEN    *cdkscreen;
    CDKLABEL     *demo;
    WINDOW       *screen;
-   char         *mesg[4];
+   char         *mesg[5];
 
    /* Initialize the Cdk screen.  */
    screen = initscr();
    cdkscreen = initCDKScreen (screen);
 
+   /* Start CDK Colors */
+   initCDKColor();
+
    /* Set the labels up.  */
-   mesg[0] = "<R></B/1>This line should have a yellow foreground and a blue background.<!1>";
-   mesg[1] = "</U/2>This line should have a white  foreground and a blue background.<!2>";
+   mesg[0] = "<R></B/31>This line should have a yellow foreground and a blue background.<!31>";
+   mesg[1] = "</U/05>This line should have a white  foreground and a blue background.<!05>";
    mesg[2] = "<B=+>This is a bullet.";
    mesg[3] = "<I=10>This is indented 10 characters.";
    mesg[4] = "<C>This line should be set to whatever the screen default is.";
@@ -295,7 +304,7 @@
 .ce
 \fI----------------------------------------\fR
 .nf
-#include "cdk.h"
+#include <cdk.h>
 
 void main()
 {
@@ -313,9 +322,9 @@
    initCDKColor();
 
    /* Set the labels up.  */
-   mesg[0] = "<C><#UL><#HL(25)><#UR>";
+   mesg[0] = "<C><#UL><#HL(26)><#UR>";
    mesg[1] = "<C><#VL></R>This text should be boxed.<!R><#VL>";
-   mesg[2] = "<C><#LL><#HL(25)><#LR>";
+   mesg[2] = "<C><#LL><#HL(26)><#LR>";
    mesg[3] = "<C>While this is not.";
 
    /* Declare the labels.  */
